### PR TITLE
v1.3.2

### PR DESCRIPTION
**Pull request by tmeyer2115**
# Version 1.3.2

## Changes

* The SDK's version is now computed automatically, instead of being hard-coded in constants.js.

## Fixes

* The behavior of ```Answers.init``` changed when the ```Answers Status``` check was added to it. Specifically, the SDK's initialization became asynchronous. Prior to this change, invoking ```init``` would immediately initialize the internals of the library. The AEB is dependent on this synchronous behavior. By fixing the status check to block only ```onReady```, this patch ensures the desired, synchronous behavior. 
* Fixed incorrect margin styling for SDK cards.
* Corrected minor styling for the AccordionCard.
* Added a needed null check in the AccordionCard's Handlebars template.

### CODE REVIEW

File: src/ui/templates/cards/accordion.hbs [hunk 1]
```diff
@@ -1,26 +1,36 @@
...
-    <div class="yxt-AccordionCard-icon js-yxt-AccordionCard-icon{{#if isExpanded}} yxt-AccordionCard-icon--expanded{{/if}}"
-      data-component="IconComponent"
-      data-opts='{"iconName": "chevron"}'
-      data-prop="icon">
+  {{/if}}
+
+  {{#if _config.details}}
+    <div class="yxt-AccordionCard-body js-yxt-AccordionCard-body" aria-hidden="{{#if isExpanded}}false{{else}}true{{/if}}">
+      <div class="yxt-AccordionCard-content">
+        <div class="yxt-AccordionCard-subtitle">{{{_config.subtitle}}}</div>
```
Comment (creotutar) [inline src/ui/templates/cards/accordion.hbs:28]: I realize that what I am asking is a breaking change. Maybe we want an item for this for later? config.subtitle and config.title, are they guaranteed to be available? Do we want to check for their existence before showing the html (which have margin/padding styling)?

Comment (tmeyer2115) [reply src/ui/templates/cards/accordion.hbs:28]: It's been unguarded like that for a while. Made an item to address in a subsequent release: https://yexttest.atlassian.net/browse/SPR-2445

**Review by alexisgrow — APPROVED** [review]
LGTM

**Review by creotutar — APPROVED** [review]
🚀 🚀 🚀 LGTM

**Pull request merged**